scripts/jobs/customer_services/vonage_landing_to_raw.py

```diff
@@ -1,60 +1,218 @@
-import sys
-from awsglue.transforms import *
-from awsglue.utils import getResolvedOptions
 from pyspark.context import SparkContext
 from awsglue.context import GlueContext
-from awsglue.job import Job
-from pyspark.sql.functions import col, lit, when, concat
-from pyspark.sql.types import IntegerType
+
 from pyspark.sql import SparkSession
+
+
 import boto3
-import pyspark.sql.functions as F
+import re
+import pandas as pd
+import json
+
+from pyspark.sql.types import StructType, StructField, StringType, IntegerType
+from pyspark.sql.functions import lit
 
-from scripts.helpers.helpers import get_glue_env_var, add_import_time_columns, get_s3_subfolders, PARTITION_KEYS, \
-    clean_column_names, get_max_date_partition_value_from_glue_catalogue
+from scripts.helpers.helpers import PARTITION_KEYS
+
+# Base Loads
 
 s3_client = boto3.client('s3')
+
+landing_zone_bucket = "dataplatform-stg-landing-zone"
+raw_zone_bucket = "dataplatform-stg-raw-zone"
+prefix = "customer-services/manual/vonage"
```

Comment:
these should be env variables - they'll be different in prod, stg and dev

Reply:
Yeah thats true ill update these now to accept env variables from glue